.eslintrc.js

```diff
@@ -18,7 +18,6 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:ember/recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
I don't think this actually existed, at least it didn't seem to enable any rules